src/components/ContentRowButtonPlusMenu.jsx

```diff
@@ -116,7 +102,6 @@ function ContentRowButtonPlusMenu({repoInfo, reposModCount, setReposModCount, is
 
     useEffect(() => {
         if (contentRowOpen === true) {
```

Comment:
You don't need the test since contentRowOpen is a boolean, you just need `(contentRowOpen)`. Do a search for `=== true` and `=== false`